src/lxp/packet.rs

```diff
@@ -822,7 +824,7 @@ impl TranslatedData {
     fn decode(input: &[u8]) -> Result<Self> {
         let len = input.len();
         if len < 38 {
-            bail!("packet too short");
+            bail!("TranslatedData::decode packet too short");
```

Comment:
These prefixes were handy when I was debugging the 41-register (and 178-register) `read_hold` messages :sweat_smile:

Reply:
👍